sdcm/cluster.py

```diff
@@ -1839,6 +1840,15 @@ def config_setup(self,
 
             if ldap:
                 scylla_yml.update(self.get_ldap_config())
+            elif ms_ad_ldap:
+                scylla_yml.update({
+                    'ldap_attr_role': 'cn',
+                    'ldap_bind_dn': 'cn=admin,dc=scylla-qa,dc=com',
```

Comment:
@fgelcer please check that the documentation has this info, at least as an example.

Reply:
checking

Reply:
@lauranovich , do we have any documentation about LDAP feature connecting to MS Active Directory server? I couldn't find any....

Reply:
@fgelcer the existing documentation applies to any LDAP server.

Reply:
@roydahan see https://github.com/scylladb/scylla-docs/blame/master/operating-scylla/security/ldap-authorization.rst#L100